tests/utils.py

```diff
@@ -23,3 +21,38 @@ def error(msg):
 def success(msg):
     """Print a success msg: Green color."""
     print("\033[92m{}\033[0m".format(msg))
+
+
+def space_check(string):
+    """Checks if a string contains an empty space."""
+    if " " in string:
+        error("'{}' contains an empty space".format(string))
+        return True
+    else:
+        return False
+
+
+def sort_check(list, stype, root=""):
+    """
+    Checks if a list of 'stype' strings is sorted, case insensitive. Takes as
+    optional string 'root' as a root value for use with sublists.
+    """
+    has_errors = False
+    reference = sorted(list, key=lambda item: item.lower())
+
+    for i in range(len(list)):
+        if list[i] == reference[i]:
+            continue
+
+        correct_index = reference.index(list[i])
+        has_errors = True
+
+        if root == "":
+            error_msg = "{} '{}' not correctly ordred"
+        else:
+            error_msg = "{} '{}' of '{}' not correctly ordred"
+
+        error(error_msg.format(stype, list[i], root))
```

Comment:
Extraneous `format` arguments are simply ignores so using one `error` for both messages fine.